### PR TITLE
Custom Elements api v1

### PR DESCRIPTION
**Pull request by midorikocak**
see: https://developers.google.com/web/fundamentals/getting-started/primers/customelements

### CODE REVIEW

**Comment by mttmccb** [comment]
Would be good to get this up to date, what's stopping it?

**Review by john-raymon — APPROVED** [review]

**Comment by midorikocak** [comment]
who can merge it?

File: my-element.html [hunk 1]
```diff
@@ -1,23 +1,27 @@
...
+        }
 
-    // Fires when an instance was removed from the document
-    element.detachedCallback = function() {};
+        attributeChangedCallback(attrName, oldVal, newVal) {
+        }
 
-    // Fires when an attribute was added, removed, or updated
-    element.attributeChangedCallback = function(attr, oldVal, newVal) {};
+        // Fires when an attribute was added, removed, or updated
```
Comment (robdodson) [inline my-element.html]: I think this comment needs to go above line 17

File: my-element.html [hunk 2]
```diff
@@ -1,23 +1,27 @@
...
+        adoptedCallback() {
```
Comment (robdodson) [inline my-element.html]: Need to add a comment that states that this fires when an element is moved to a new document.

**Review by robdodson — CHANGES_REQUESTED** [review]

File: my-element.html (hunk 2 above)
Comment (midorikocak) [reply my-element.html]: Addingn

File: my-element.html (hunk 1 above)
Comment (midorikocak) [reply my-element.html]: Done

File: my-element.html (hunk 2 above)
Comment (midorikocak) [reply my-element.html]: Done

**Comment by midorikocak** [comment]
Done. Can you review once more?

**Review by robdodson — APPROVED** [review]

**Comment by robdodson** [comment]
Thanks @midorikocak. Could you sign the CLA so we can land the change?

**Comment by midorikocak** [comment]
Hi, I've signed it. Thanks.

**Pull request merged**